### PR TITLE
Update Discord links

### PR DESCRIPTION
**Pull request by brokenthumbs**
## Description

Reference: https://discord.com/channels/684494798358315010/947551295546462278/1200253124137455709

- The Discord link in the footer currently leads to a compromised Discord server.
  - If you check the Discord at https://discord.gg/regen-network, you will find a phishing verification bot.
  - This is a problem for anybody who is not savvy enough to spot a phishing attempt.
- This PR updates the link to the actual Discord, which is referenced here: https://guides.regen.network/guides/regen-network/network-basics/connect-with-us
- Is there anything that can be done to delete or terminate the compromised Discord server at https://discord.gg/regen-network ?
  - Please strongly consider how to terminate the old compromised Discord server at https://discord.gg/regen-network

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

1. Confirm that there is indeed a phishing verification bot in the current footer Discord link at https://discord.gg/regen-network
2. Confirm that the new Discord link, https://discord.gg/TbRfsNQGWH, leads to the actual Regen Discord server, as referenced by https://guides.regen.network/guides/regen-network/network-basics/connect-with-us

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by blushi — APPROVED** [review]
Thanks for your contribution!
We've reached to discord regarding the wrong regen-network server.
Pre-approving. Could you rebase your work on dev?

File: web-components/src/components/footer/SocialLinks.tsx [hunk 1]
```diff
@@ -120,7 +120,7 @@ const SocialLinks: React.FC<React.PropsWithChildren<Props>> = ({
           <GithubIcon className={styles.icon} />
         </Link>
         <Link
-          href="https://discord.gg/regen-network"
+          href="https://discord.gg/TbRfsNQGWH"
```
Comment (clevinson) [inline web-components/src/components/footer/SocialLinks.tsx:123]:
```suggestion
          href="https://discord.gg/6ghMcMBjbz"
```

File: web-www/components/templates/press-kit/ConnectSection/ConnectSection.config.tsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ export const getPressKitConnectIcons = (theme: Theme): IconLabelProps[] => [
         hoverColor={theme.palette.secondary.main}
       />
     ),
-    href: 'https://discord.gg/BDcBJu3',
+    href: 'https://discord.gg/TbRfsNQGWH',
```
Comment (clevinson) [inline web-www/components/templates/press-kit/ConnectSection/ConnectSection.config.tsx:62]:
```suggestion
    href: 'https://discord.gg/6ghMcMBjbz',
```

File: web-www/components/templates/token/TokenConnectSection/TokenConnectSection.config.tsx [hunk 3]
```diff
@@ -27,7 +27,7 @@ export const getIcons = ({ theme }: Params): IconLabelProps[] => [
         hoverColor={theme.palette.secondary.main}
       />
     ),
-    href: 'https://discord.gg/BDcBJu3',
+    href: 'https://discord.gg/TbRfsNQGWH',
```
Comment (clevinson) [inline web-www/components/templates/token/TokenConnectSection/TokenConnectSection.config.tsx:30]:
```suggestion
    href: 'https://discord.gg/6ghMcMBjbz',
```

**Review by clevinson — CHANGES_REQUESTED** [review]
Thanks for the report and updating this!

The invite link in this PR currently directs people to the #activity-log, and I think it'd actually be better to direct people to the #welcome channel first.  I've just created a new invite link and suggested you update it here:

https://discord.gg/6ghMcMBjbz

**Comment by brokenthumbs** [comment]
Updated Discord link in pull request!

@clevinson You may also want to update the Discord link here as well: https://guides.regen.network/guides/regen-network/network-basics/connect-with-us

I don't know where that source code is for that particular FAQ page is.

**Comment by brokenthumbs** [comment]
On another note, how did somebody take over that `discord.gg/regen-network` shortcut link?

**Comment by dpdanpittman** [comment]
I would also like to know this. It seems like that shouldn't be possible.

**Review by clevinson — APPROVED** [review]

**Comment by clevinson** [comment]
@brokenthumbs @dpdanpittman I'm pretty sure that we just stopped paying for the feature (discord charges for you to have custom invite links) and forgot to change it on our website.  Then after a while, a hacker just created a new discord server and squatted / paid for the custom URL that we had used in the past.

**Comment by dpdanpittman** [comment]
That makes sense.

**Comment by brokenthumbs** [comment]
How long does the build take?
Thanks for approving!

Hope we can take down the bad actor Discord.
Or at least not send people there.

**Pull request merged**